docs/api.md

```diff
@@ -1158,6 +1161,9 @@ If there's no element matching `selector`, the method throws an error.
 
 Shortcut for [page.mainFrame().focus(selector)](#framefocusselector).
 
+#### page.frameManager()
```

Comment:
Let's not expose the frameManager and networkManager in the public API.

Reply:
If these methods are not exposed, we have to call private methods from outside of the class, which breaks the encapsulation principle of OOP. I'm fine with it though - you're more familiar with the codebase, so you probably know how good the rest of the code fares in this respect.
I decided to ditch getter methods altogether since we're accessing private stuff anyway, and just stick to the properties.